util/php2po.php

```diff
@@ -20,52 +33,174 @@ class TmpA {
 		return;
 	}
 
+	// utility functions
+	function startsWith($haystack, $needle) {
+		// search backwards starting from haystack length characters from the end
+		return $needle === "" || strrpos($haystack, $needle, -strlen($haystack)) !== FALSE;
+	}
+
+
+	// start !
 	include_once($phpfile);
 
-	print "Out to '$pofile'\n";
+	$out  = '';
+	$out .= "# FRIENDICA Distributed Social Network\n";
+	$out .= "# Copyright (C) 2010, 2011, 2012, 2013 the Friendica Project\n";
+	$out .= "# This file is distributed under the same license as the Friendica package.\n";
+	$out .= "# \n";
+	$out .= 'msgid ""' ."\n";
+	$out .= 'msgstr ""' ."\n";
+	$out .= '"Project-Id-Version: friendica\n"' ."\n";
+	$out .= '"Report-Msgid-Bugs-To: \n"' ."\n";
+	$out .= '"POT-Creation-Date: 2015-09-22 09:58+0200\n"' ."\n";
+	$out .= '"PO-Revision-Date: 2015-10-06 17:43+0000\n"' ."\n";
+	$out .= '"Language-Team: Italian (http://www.transifex.com/Friendica/friendica/language/it/)\n"' ."\n";
```

Comment:
Is the language team an optional header info? Or could it be more general?

Also the creation date-time should not be fix I think.